requirements.txt

```diff
@@ -1,3 +1,5 @@
+cherrypy
 codeclimate-test-reporter
 coverage
+nose
```

Comment:
@jaraco cherrypy/cherrypy#1463 only states, that the framework itself relies on `nose`, but actual web apps written on top of it do not. And from my own experience tests in my apps, which get stuck forever (for unknown reason) when running nose testrunner, succeed with just changing it to `py.test`.